exercises/content-curation.md

```diff
@@ -0,0 +1,28 @@
+### Exercise
+
+This exercise is to assess your detective skills, and your content curation skills. It is similar to a typical task you would work on at Snyk.
+
+You are given a CVE ID and asked to discover all information on it. You do not need prior knowledge of security exploits, but it may help.
+
+You can use any external source available on the internet, NOT including Snyk.io .
+
+#### Tasks
+A large company, Equifax, was breached by hackers on March 13th, 2017. They used a known exploit, assigned the ID: **CVE-2017-5638**.
```

Comment:
I think the breach happened later, no? Maybe mention the disclosure Equifax made on September 8th, and ask to backtrack from there?